components/TextFields/src/MDCTextInputControllerLegacyDefault.m

```diff
@@ -164,7 +164,7 @@ - (UIOffset)floatingPlaceholderOffset {
   }
 
   CGFloat horizontal =
-      placeholderWidth * (1 - (CGFloat)self.floatingPlaceholderScale.floatValue) * .5f;
+      placeholderWidth * (1 - (CGFloat)self.floatingPlaceholderScale.floatValue) * (CGFloat).5;
```

Comment:
tip: for consistency, let's make it `(CGFloat)0.5`

Reply:
Done throughout.